snapcraft/how-to/how-to-use-snap-review.md

```diff
@@ -0,0 +1,97 @@
+# How to use the snap review-tools to verify your snap
+
+`review-tools` is a tool that checks whether a snap meets the requirements for the Snap Store. Once you have created a snap it is highly recommended to use `review-tools` to verify it before submitting it to the Store. 
+
+This guide shows how to use `review-tools`, and considers some of its commands.
+
+## Prerequisites
+
+This guide assumes:
+- Command line knowledge
+- A Linux system running Ubuntu
+- An already built snap
+
+## Enable snapd
+
+First check whether `snapd` is enabled on your system. If you are using Ubuntu 16.04 LTS (Xenial Xerus) or later you most likely have `snapd` installed. 
+
+In the terminal type:
+
+```bash
+snap version
+```
+
+If you have `snapd` you will get something that looks like:
+
+```no-highlight
+snap    2.63+22.04
+snapd   2.63+22.04
+series  16
+ubuntu  22.04
+kernel  6.5.0-41-generic
+```
+
+If you do not have _snap_ it can be installed in two ways:
+
+1. Search for _snapd_ on the Ubuntu Software Centre and install
+2. Using the command line:
+
+```bash
+sudo update
+sudo apt install snapd
+```
+
+Then either log out and back in again, or restart your system. 
```

Comment:
I'm not convinced this part is necessary; I think it might be reasonable to assume that any (developer) who has built a snap used snapcraft to do so, which means they already have snapd installed and running.
Maybe linking to [this](https://snapcraft.io/docs/installing-snapd) page as a \<note\> is sufficient?